backend/app/GraphQL/Mutations/ToggleBookmark.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\GraphQL\Mutations;
+use App\Models\Answer;
+use App\Models\Question;
+use App\Models\Bookmark;
+use Exception;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Str;
+final class ToggleBookmark
+{
+    /**
+     * @param  null  $_
+     * @param  array{}  $args
+     */
+    public function __invoke($_, array $args)
+    {
+        try {
+
+            if($args['bookmarkable_type'] == "Answer") {
+                $bookmarkable = Answer::findOrFail($args['bookmarkable_id']);
+            }
+
+            if($args['bookmarkable_type'] == "Question") {
+                $bookmarkable = Question::findOrFail($args['bookmarkable_id']);
+            }
+
+            if ($bookmarkable->bookmarks()->whereUserIdAndBookmarkableIdAndBookmarkableType($args['user_id'], $args['bookmarkable_id'], "App\Models\\".$args['bookmarkable_type'])->delete()){
```

Comment:
```suggestion
            if ($bookmarkable->bookmarks()->whereUserId($args['user_id'])->delete()){
```

Because you already queried for the bookmarkable you don't need to repeat querying for it